src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -265,9 +267,22 @@
 
     @Override
     public void onTraversalStart() {
+        //check references
+        final SAMSequenceDictionary mainDict = getBestAvailableSequenceDictionary();
```

Comment:
Is there any reason why this check is particular to `CalculateGenotypePosteriors`?  It seems like it could be extracted and used with every `List<FeatureInput<VariantContext>>` in the GATK.

Reply:
Also, do we need to be as strict as `isSameDictionary()`?  Maybe it's enough just to have the same sequence dictionary when subset to the contigs used for traversal.